segment/segment.py

```diff
@@ -37,7 +39,7 @@ def scale_frames(self, frames, scale_x=4, scale_y=4):
 
     # Combine respective frames into a video
     def frames_to_video(self, frames, fps):
-        print('Combining frames')
+        logging.info('Combining frames')
         fourcc = cv2.VideoWriter_fourcc('D', 'I', 'V', 'X')
         out = cv2.VideoWriter(self.__path_out, fourcc, fps, frames[0].shape[:2][::-1])
```

Comment:
Why is this being done in this portion?
```
frames[0].shape[:2][::-1]
```

For such portion, add comment as it might not be clear to someone reading your code. I know [::-1] does reverse of the list and [:2] takes first two elements, but why is this being done? Comments should answer the *why* rather than "*what*".